tests/config/junos/CONFIGURATION_ERROR/default/yang.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
We might revisit this structure later (I can't think of a better structure currently though), but I fear it won't be compatible with other platforms. That's just a FYI.

Reply:
I agree this will not last in the long run. Ideally I think it should map to actual openconfig models depending on the error. I can't even begin to imagine what that would look like in a mapping definition though (outside of some complex python version).

Reply:
Discussion to follow under https://github.com/napalm-automation/napalm-logs/issues/175